cms/server/admin/handlers/contestuser.py

```diff
@@ -163,6 +163,8 @@ def post(self, contest_id, user_id):
             attrs = participation.get_attrs()
 
             self.get_string(attrs, "password")
+            if attrs["password"] == "":
+                attrs["password"] = None
```

Comment:
I think you can use get_string with empty=None - can you please check and amend the commit if true? Thank you Edoardo!